src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/events/PluginPatchsetCreatedEvent.java

```diff
@@ -109,6 +112,10 @@ public boolean shouldTriggerOn(GerritTriggeredEvent event) {
         if (!super.shouldTriggerOn(event)) {
             return false;
         }
+        // if it was a manually triggered event, always honor it
```

Comment:
You can't setup a job to trigger *only* on drafts (at least currently), can you ?
But indeed you could have a "light" job that always triggers, and a "heavier" that runs only on non-drafts, and if someone manually triggers, you can't guess if it's meant to (re)trigger the normal "light" job, or to exceptionally launch the "heavy" one.
Actually, I initially wrote the patch without skipping the draft check, but then thought it was probably better to have *some* way of running a "non-draft" job on a draft (instead of *no* way), even at the cost of an occasional unintended run.
I can of course narrow the feature to "trivial rebases/no code change" if you think it desirable, but I'm afraid I'm not familiar enough with the GUI handling to find the time to have it configurable.